components/omega/test/ocn/TendencyTermsTest.cpp

```diff
@@ -1029,13 +1022,23 @@ void initTendTest(const std::string &MeshFile, int NVertLevels) {
       ABORT_ERROR("TendencyTermsTest: error initializing default halo");
    }
 
+   VertCoord::init();
+
+   // Reset NVertLayers to the test value
+   auto *DefVertCoord        = VertCoord::getDefault();
+   DefVertCoord->NVertLayers = NVertLayers;
+   Dimension::destroy("NVertLayers");
+   std::shared_ptr<Dimension> VertDim =
+       Dimension::create("NVertLayers", NVertLayers);
```

Comment:
I think a block like this is also needed in `AuxiliaryVarsTest.cpp`. It fails without it.